checklist-service/test/services/checklists/checklist.test.js

```diff
@@ -164,11 +198,41 @@ test('list all checklists', async t => {
 
   const response = await checklist.list(record)
 
-  t.same(response, testLists)
+  t.same(response, testLists[userId])
   t.equal(
     received.dynamoDb.query.ExpressionAttributeValues[':userId'],
     record.userId
   )
 
   t.end()
 })
+
+test('list all checklists including shared lists', async t => {
+  const checklist = require('../../../services/checklists/checklist')
+
+  const record = {
+    userId: 'ownerB'
+  }
+
+  const response = await checklist.list(record)
+
+  t.match(response, testLists['ownerB'])
+
```

Comment:
The part of the test that checks that the testLists has the correct the name and description in the shared list is missing